.travis.yml

```diff
@@ -5,31 +5,40 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
 env:
-  - SYMFONY_VERSION=2.3.*
+  - SYMFONY_VERSION=2.7.*
 
 matrix:
-  allow_failures:
-    - env: SYMFONY_VERSION=dev-master
   include:
-    - php: 5.5
+    - php: 5.6
       env: SYMFONY_VERSION=2.3.*
-    - php: 5.5
+    - php: 5.6
       env: SYMFONY_VERSION=2.4.*
-    - php: 5.5
+    - php: 5.6
       env: SYMFONY_VERSION=2.5.*
-    - php: 5.5
+    - php: 5.6
       env: SYMFONY_VERSION=2.6.*
-    - php: 5.5
-      env: SYMFONY_VERSION=2.7.*
-    - php: 5.5
-      env: SYMFONY_VERSION='2.8.*@dev'
+    - php: 5.6
+      env: SYMFONY_VERSION=2.8.*@dev
+    - php: 5.6
+      env: SYMFONY_VERSION=3.0.*@dev
+    - php: 5.3
+      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
changing your Symfony constraint while testing lowest deps does not make sense, as you also change the lowest bound